gradle/libs.versions.toml

```diff
@@ -11,6 +11,7 @@ publish = "0.35.0"
 hibernate = "7.2.0.Final"
 springOrm = "7.0.2"
 springTx = "7.0.2"
+springAspects = "7.0.2"
```

Comment:
`springOrm`, `springTx`, and `springAspects` are all the `spring` version. We can consider merging them into one version.